ui/src/components/CandidateList/PlayerTimeDisplay.tsx

```
@@ -0,0 +1,107 @@
+                      background: "rgba(0,0,0,.9)",
+                      borderRadius: "4px",
+                      color: "white",
+                    }}
+                  >
+                    <span>{detectionTime}</span>
+                    {" " + d.newCategory + " "}
+                    {d.comments}
+                  </mark>
+                </ListItem>
+              );
+            })}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)